homeassistant/components/roomba/sensor.py

```diff
@@ -114,6 +119,20 @@ class RoombaSensorEntityDescription(
         value_fn=lambda self: self.run_stats.get("nScrubs"),
         entity_registry_enabled_default=False,
     ),
+    RoombaSensorEntityDescription(
+        key="total_cleaned_area",
+        translation_key="total_cleaned_area",
+        icon="mdi:texture-box",
+        native_unit_of_measurement=AREA_SQUARE_METERS,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda self: (
+            self.run_stats.get("sqft") * 9.29
+            if self.run_stats.get("sqft") is not None
```

Comment:
Use a walrus and store the value in a local variable so we don't need to look it up twice.